clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -2882,6 +2882,82 @@ raw2trace_t::set_instr_summary_flags(raw2trace_thread_data_t *tdata, uint64 modi
     return true;
```

Comment:
Seems like #5036 should be referenced in the description?

Reply:
Done.